src/tools/pub/cmd/pub-global.md

```diff
@@ -211,49 +211,60 @@ Use `list` to list all currently active packages.
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
-`<constraint>`
-: Optional for `dart pub global activate`. The constraint allows you to pull
-  in a specific version of the package. For example,
-  the following command pulls the 0.6.0 version of the `markdown`
-  package:
-
-  ```terminal
-  $ dart pub global activate markdown 0.6.0
-  ```
-
-  If you specify a range, pub picks the best version that meets that
-  constraint. For example:
-
-  ```terminal
-  $ dart pub global activate foo <3.0.0
-  ```
-
-`--executable=<name>` or `-x<name>`
-: Optional for `dart pub global activate`.
-  Adds the specified executable to your PATH.
-  You can pass more than one of these flags.
-  For example, the following command adds `bar` and `baz` (but not
-  any other executables that `foo` might define) to your PATH.
-
-  ```terminal
-  $ dart pub global activate foo -x bar -x baz
-  ```
-
-`--no-executables`
-: Optional for `dart pub global activate`.
-  Globally activates the package but doesn't put any
-  executables in `bin`. You have to use `dart pub global run` to
-  run any executables.
-
-`--overwrite`
-: Optional for `dart pub global activate`.
-  Normally, if executables from two global packages have a name
-  collision, the preexisting executable wins. If you specify this flag,
-  the new executable overwrites the previously activated executable.
-
-<aside class="alert alert-info" markdown="1">
-  *Problems?* See [Troubleshooting pub](/tools/pub/troubleshoot).
-</aside>
+### `<constraint>`
+
+Optional for `dart pub global activate`. 
```

Comment:
As before, I'm not crazy about this on a line by itself. In general, I'd like to avoid multiple single-sentence/line paragraphs in a row. So everything I said for that other page (whatever it was) applies here (and for the rest of the options on this page), too.